src/game_board.cpp

```diff
@@ -182,6 +182,24 @@ void game_board::check_victory(bool& continue_level,
 	continue_level = false;
 }
 
+void game_board::set_affect_distant_max_radius(std::optional<int> value)
```

Comment:
Should use `utils::optional` rather than `std::optional`.

Reply:
> Should use `utils::optional` rather than `std::optional`.

`std::optional` replaced by  `utils::optional` everywhere.